tests/baseTestAPI.py

```diff
@@ -117,7 +120,7 @@ def setUpClass(cls):
 
         # Set the Agency for the agency user
         agencyUser = userDb.getUserByEmail(test_users['agency_user'])
-        agencyUser.agency = "testAgency"
+        agencyUser.cgac_code = "testAgency"
```

Comment:
Since real-life CGAC codes are 4 digit numbers, does it make more sense to use something like that for our unit test data for better parallels to what we'd see in actual use?